### PR TITLE
Refine activity log events for moped project notes

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/11291

Something that Quill does as a part of its rich text editing i believe is enclose every comment with a `<p>`. So showing the comment text results in each comment/note being on a separate line from the description. ~~I indented the styling some, but of course am open to different ways to display the information.~~ And I'm back to indenting the comment!

Screen Shot 2023-01-27 at 11 51 24 AM

## Testing
**URL to test:** 
https://deploy-preview-956--atd-moped-main.netlify.app/moped/projects/1695?tab=activity_log

**Steps to test:**

Add a status update (from summary page and/or dashboard)
Add a comment (from comments tab)

Edit a status update
Edit a comment

delete a status update
delete a comment

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/utils/activityLogHelpers.js [hunk 1]
```diff
@@ -30,11 +28,17 @@ export const formatActivityLogEntry = (change, lookupData) => {
     case "moped_proj_tags":
       return formatTagsActivity(change, lookupData.tagList);
     case "moped_proj_funding":
-      return formatFundingActivity(change, lookupData.fundingSources, lookupData.fundingPrograms);
+      return formatFundingActivity(
+        change,
+        lookupData.fundingSources,
+        lookupData.fundingPrograms
+      );
```
Comment (chiaberry) [inline moped-editor/src/utils/activityLogHelpers.js:35]: just linting

File: moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js [hunk 2]
```diff
@@ -674,68 +619,10 @@ export const ProjectActivityLogCreateDescriptions = {
...
-    label: (record, userList) => {
-      //return '"' + record.record_data.event.data.new.funding_description + "\" as a new funding source";
-      return (
-        "A new funding source" +
-        (record.record_data.event.data.new.funding_description
-          ? ": " + record.record_data.event.data.new.funding_description
-          : "")
-      );
-    },
-  },
```
Comment (tillyw) [inline moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js:738]: thanks for cleaning up this file! ✨ i'll be more consistent about this in my future prs!!

File: moped-editor/src/utils/activityLogFormatters/mopedNotesActivity.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  if (change.description.length === 0) {
+    return {
+      changeIcon,
+      changeText: [
+        {
+          text: noteType === 2 ? "Added a status update " : "Added a comment ",
+          style: "boldText",
+        },
+        {
+          text: parse(change.record_data.event.data.new.project_note),
```
Comment (tillyw) [inline moped-editor/src/utils/activityLogFormatters/mopedNotesActivity.js:18]: it makes sense to use the parser here!

**Review by tillyw — APPROVED** [review]
this looks great!! i have some suggestions for possible usability improvements.
Screen Shot 2023-01-31 at 4 33 59 PM
would we be open to including the comment text with the 'edit' and 'delete' actions like we do with the 'added' actions. i think knowing which comment was edited or deleted might be helpful! i understand it might clog up the activity log if the comments are long, but maybe we can take the first two lines or so and truncate the rest with an ellipsis?

File: moped-database/metadata/tables.yaml [hunk 4]
```diff
@@ -2760,24 +2760,38 @@
...
+          value_from_env: ACTIVITY_LOG_API_SECRET
+      request_transform:
+        body:
+          action: transform
+          template: |-
+            {
+              "query": "mutation InsertActivity($object: moped_activity_log_insert_input!, $project_id:Int!, $updated_at:timestamptz ) { insert_moped_activity_log_one(object: $object) { activity_id } update_moped_project_by_pk(pk_columns: {project_id: $project_id}, _set: {updated_at: $updated_at}) { updated_at }}",
+              "variables": {
+                "object": {
+                  "record_id": {{ $body.event.data.new.project_note_id }},
```
Comment (chiaberry) [inline moped-database/metadata/tables.yaml:2780]: @johnclary i fixed this since you last looked at it

Comment (johnclary) [reply moped-database/metadata/tables.yaml:2780]: ✅ looks great 🙏

Comment (johnclary) [reply moped-database/metadata/tables.yaml:2780]: i will make an issue to address this in our other triggers

**Pull request merged**